site/content/en/docs/concepts/topology_aware_scheduling.md

```diff
@@ -98,14 +98,35 @@ to a TAS ResourceFlavor.
 
 {{< include "examples/tas/sample-job-preferred.yaml" "yaml" >}}
 
+### ClusterAutoscaler support
+
+TAS integrates with the [Kubernetes ClusterAutoscaler](https://github.com/kubernetes/autoscaler/tree/master/cluster-autoscaler)
+through the [Provisioning AdmissionCheck](/docs/admission-check-controllers/provisioning/).
+
+When a workload is assigned to the TAS Resource Flavor with Provisioning
+AdmissionCheck, then its admission flow has the following stages:
+1. **Quota reservation**: quota is reserved, and the workload obtains the
+  `QuotaReserved` condtion. Preemptions are evaluated if configured.
+2. **Admission checks**: waiting for all AdmissionChecks, including the
+  Provisioning one, to report `Ready`.
+3. **Topology assignment**: TAS, in the second pass of the Kueue scheduler,
+  calculates available capacity, including any newly provisioned nodes, and
+  updates the assignment with the assignment of topology domains. Once done the
+  `Admitted` condition is set on the Workload.
+
+Check also [PodSet updates in ProvisioningRequestConfig](site/content/en/docs/admission-check-controllers/provisioning.md)
+to see how you can configure Kueue if you want to restrict scheduling to the
+newly provisioned nodes (assuming the provisioning class supports it).
+
 ### Limitations
 
 Currently, there are limitations for the compatibility of TAS with other
-features. In particular, a ClusterQueue referencing a TAS Resource
-Flavor (with the `.spec.topologyName` field) is marked as inactive if also
-using [MultiKueue](multikueue.md) or
-[ProvisioningRequest](/docs/admission-check-controllers/provisioning/) admission
-checks.
+features, including:
+- some scheduling directives (e.g. pod affinities and anti-affinities) are ignored,
+- the "podset-required-topology" annotation may fail if the underlying
+  ClusterAutoscaler cannot provision nodes that satisfy the domain constraint,
+- a ClusterQueue referencing a TAS Resource Flavor (with the `.spec.topologyName`
+field) is marked as inactive if also using [MultiKueue](multikueue.md).
```

Comment:
```suggestion
- a ClusterQueue for [MultiKueue](multikueue.md) references ResourceFlavors with Topology name (`.spec.topologyName`) is marked as inactive.
```

Does this mean that CQ for MultiKueue does not support TAS, right?

Reply:
Correct, currently no. We have an issue to support it: https://github.com/kubernetes-sigs/kueue/issues/5313

Reply:
Done